pkg/cmd/create/install_dependencies.go

```diff
@@ -37,19 +38,11 @@ var (
 		"kubectl",
 		"gcloud",
 		"helm",
-		"ibmcloud",
 		"tiller",
 		"helm3",
-		"kops",
-		"kvm",
-		"kvm2",
 		"ksync",
 		"oc",
-		"virtualbox",
-		"xhyve",
-		"hyperv",
 		"terraform",
```

Comment:
I don't think we need `terraform` either. There is a `terraform.go`, but none of the functions is used, so that could be deleted at the same time. `jx` should not call `terraform`.